### DISCUSSION TITLE
Does a SubMenu has an exitAction support?

### DISCUSSION THREAD

**Discussion by menachemp** [daniele77/cli#144] [Q&A]
Hi daniele,

is there a way to get an ExitAction for a SubMenu?
like the ExitAction available for CLI object.

Thanks

**Comment by daniele77** [comment]
Hi @menachemp ,
at the moment menus and submenus don't have exit actions.
However, it could be a nice feature to have. If you want, please fill an issue with your request.
Thank you.
